baseplate/sidecars/live_data_watcher.py

```diff
@@ -20,93 +25,36 @@
 from baseplate.lib.secrets import secrets_store_from_config
 from baseplate.server import EnvironmentInterpolation
 
-try:
-    import boto3  # type: ignore
-    from botocore.client import ClientError  # type: ignore
-
-    S3_FETCHER_ENABLED = True
-except ImportError:
-    S3_FETCHER_ENABLED = False
-
 
 logger = logging.getLogger(__name__)
 
 
 HEARTBEAT_INTERVAL = 300
 
 
+class LoaderException(Exception):
+    pass
+
+
+class LoaderType(Enum):
+    PASSTHROUGH = "PASSTHROUGH"
+    S3 = "S3"
+
+    @classmethod
+    def _missing_(cls, value: object) -> "LoaderType":
```

Comment:
I don't like this type [but](https://github.com/python/typeshed/blob/master/stdlib/enum.pyi#L176)